### PR TITLE
Remove application_number_replacement feature flag uses

### PR DESCRIPTION
**Pull request by SamJamCul**
## Changes proposed in this pull request

https://trello.com/c/VjYLj2Tb/5033-remove-the-applicationnumberreplacement-feature-flag
Removing the `application_number_replacement` feature flag

### CODE REVIEW

File: ministerial-report-applications-2022-04-20 11:51:21 +0100.txt [hunk 1]
```diff
@@ -0,0 +1 @@
...s=>0, :offer_received=>0, :accepted=>0, :application_declined=>0, :application_rejected=>0, :application_withdrawn=>0}, {:subject=>:physics, :applications=>1, :offer_received=>0, :accepted=>0, :application_declined=>0, :application_rejected=>0, :application_withdrawn=>1}, {:subject=>:religious_education, :applications=>0, :offer_received=>0, :accepted=>0, :application_declined=>0, :application_rejected=>0, :application_withdrawn=>0}, {:subject=>:stem, :applications=>6, :offer_received=>0, :accepted=>0, :application_declined=>0, :application_rejected=>4, :application_withdrawn=>2}, {:subject=>:ebacc, :applications=>12, :offer_received=>2, :accepted=>2, :application_declined=>0, :application_rejected=>5, :application_withdrawn=>4}, {:subject=>:primary, :applications=>7, :offer_received=>3, :accepted=>2, :application_declined=>0, :application_rejected=>0, :application_withdrawn=>0}, {:subject=>:secondary, :applications=>17, :offer_received=>5, :accepted=>4, :application_declined=>1, :application_rejected=>6, :application_withdrawn=>4}, {:subject=>:total, :applications=>24, :offer_received=>8, :accepted=>6, :application_declined=>1, :application_rejected=>6, :application_withdrawn=>4}]
```
Comment (cwrw) [inline ministerial-report-applications-2022-04-20 11:51:21 +0100.txt:1]: ✂️

File: ministerial-report-applications-2022-04-20 11:49:26 +0100.txt [hunk 2]
```diff
@@ -0,0 +1 @@
...ions=>0, :offer_received=>0, :accepted=>0, :application_declined=>0, :application_rejected=>0, :application_withdrawn=>0}, {:subject=>:physics, :applications=>0, :offer_received=>0, :accepted=>0, :application_declined=>0, :application_rejected=>0, :application_withdrawn=>0}, {:subject=>:religious_education, :applications=>0, :offer_received=>0, :accepted=>0, :application_declined=>0, :application_rejected=>0, :application_withdrawn=>0}, {:subject=>:stem, :applications=>0, :offer_received=>0, :accepted=>0, :application_declined=>0, :application_rejected=>0, :application_withdrawn=>0}, {:subject=>:ebacc, :applications=>0, :offer_received=>0, :accepted=>0, :application_declined=>0, :application_rejected=>0, :application_withdrawn=>0}, {:subject=>:primary, :applications=>1, :offer_received=>0, :accepted=>0, :application_declined=>0, :application_rejected=>1, :application_withdrawn=>0}, {:subject=>:secondary, :applications=>0, :offer_received=>0, :accepted=>0, :application_declined=>0, :application_rejected=>0, :application_withdrawn=>0}, {:subject=>:total, :applications=>1, :offer_received=>0, :accepted=>0, :application_declined=>0, :application_rejected=>1, :application_withdrawn=>0}]
```
Comment (cwrw) [inline ministerial-report-applications-2022-04-20 11:49:26 +0100.txt:1]: ✂️

File: ministerial-report-applications-2022-04-20 11:51:21 +0100.txt (hunk 1 above)
Comment (SamJamCul) [reply ministerial-report-applications-2022-04-20 11:51:21 +0100.txt]: done

**Comment by cwrw** [comment]
looks good! will wait for next week 👍

**Review by cwrw — APPROVED** [review]
thanks Sam!

**Pull request merged**